database.hh

```diff
@@ -870,7 +879,20 @@ public:
     }
 
     bool compaction_enforce_min_threshold() const {
-        return _config.compaction_enforce_min_threshold;
+        return _config.compaction_enforce_min_threshold || _is_receiving_streaming;
+    }
+
+    unsigned min_compaction_threshold() {
+        // During receiving stream, (bootstrap or replace) the less we
```

Comment:
Decommission is also receiving streaming.

Reply:
It does, but it also receive reads that can be negatively affected by this.

For now I'd like to tackle just boostrap, where the risk is low.

Reply:
I have changed the names to reflect the fact that this is only used for bootstrap and replace.